pytd/tests/test_query_engine.py

```diff
@@ -8,6 +8,28 @@
 from pytd.version import __version__
 
 
+class QueryEngineEndpointSchemeTestCase(unittest.TestCase):
+    def test_presto_invalid_endpoint(self):
```

Comment:
Don't we need to have a valid case?

Reply:
added e71f1b5dba9780c526d8a5b32de2cc20324c9580